recipes/pydrift/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pydrift" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 572a5f5ee3cfde052ee858cc2ed741fbb5c3df9fbcd44a70fe1d208b751a163e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - pip
+    - catboost >=0.23,<0.24
+    - jupyter >=1.0.0,<2.0.0
+    - pandas >=1.0.3,<2.0.0
+    - plotly_express >=0.4.1,<0.5.0
+    - pytest >=5.4.2,<6.0.0
```

Comment:
Is `pytest` really a runtime requirement? Also do all of these version requirements need to be this strict?

Reply:
I´ll remove pytest, you are right

The version requirements... they are the versions I ran, I don't know it the package will break if you install another version, I know for sure that there are a couple of warnings in previous catboost versions but I don't know more about this topic :(

Reply:
Normally people only use a lower limit, i.e. `scikit-learn >=0.23.1`. Having an upper limit makes it:
* impossible to use this package with a newer version
* likely impossible to install other packages which have strict versions constraints like this

Normally upper limits are only used for compiled packages where the ABI can change. Most of the time this managed automatically using `run_exports` in packages that provide shared libraries and updates are managed using migrations (see the status and doc pages on conda-forge.org).

Reply:
Youre absolutely right! Sorry I copied it from other proposal and I let it like that, now there is only lower limit in myu dependencies :)